Lib/inspect.py

```diff
@@ -170,8 +170,10 @@ def isfunction(object):
 
 def _has_code_flag(f, flag):
     """Return true if ``f`` is a function (or a method or functools.partial
-    wrapper wrapping a function) whose code object has the given ``flag``
+    wrapper wrapping a function or a functools.partialmethod wrapping a
+    function) whose code object has the given ``flag``
```

Comment:
Shouldn't this be "or a functools.partialmethod wrapping a method"?

Reply:
At the point of wrapping it is still a function, the function is not bound.